docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/tracks.md

```diff
@@ -13,9 +13,39 @@ status: draft
+Tracks are sets of data such as genes, variants or regulatory motifs displayed along a genome. A track displays a single set of data ie genes on the forward strand.
 
-You can change which tracks you see using the Track panel on the right of the page. There are three categories of tracks: Genomic, Variation and Expression. Details of your Focus entity are always shown at the top of the track panel. Show or hide tracks using the eye icon. Find out more about a track or an entity by clicking on the ellipsis to open the drawer with more information and links to the Entity viewer. Use the arrow at the top of the Action bar to close the Track panel drawer or hide the Track panel completely.
+Individual entities ie genes, variants or regulatory regions within a track are shown on horizontal rows. There may be multiple rows within a track, ensuring that there is no entity overlap within a row. 
+
+Forward and reverse strands
+
+Some tracks, such as genes, are associated with a specific strand of the genome. Strands are represented by small arrows pointing in the direction of transcription set behind the track information. 
+
+## How to reveal the track names
+
+Opening the Track panel
+
+The track names are revealed in the Track panel. To open the Track panel select the arrow pointing right at the top of Action bar. A panel listing all Track names will appear to the right of the browser and Action bar. 
+
+There are three tabs in the Track panel, Genomic, Variation and Expression. The panel will automatically show the correct tab for your chosen Focus entity ie if your Focus entity is a gene the Genomic tab will show in the Track panel.
+
+The relevant tracks are shown in the each tab eg the Genomic tab displays tracks for the Focus gene, its transcripts, coding and non-coding protein genes on the forward and reverse strands, the reference sequence and %GC content.
+
+## How to hide and show tracks
+
+All available tracks are listed by default in the Track tabs of the Track panel.
+
+To hide a track select the eye icon to the right of the track name. The eye icon will go from blue (on) to grey (off).
+
+## How to find out more about a track 
+
+Find out more about a track or an entity by selecting the ellipsis next to the track name on the Track panel. This will open a drawer to the right of the Track panel containing more information about the track.
+
+Use the arrow at the top of the Action bar to close the Track drawer containing track information or to hide the Track panel completely.
+
+## How to reveal track and features labels
+
+You can modify the appearance of tracks by selecting on the cog icon to the right of the track on the Browser. This allows you to show or hide labels on the tracks and/or other features ie gene symbols.
```

Comment:
We'll never see the term 'gene symbol' in a cog
How about '...hide labels on the tracks or names of features, for example Transcript IDs'